sdk/purview/purview-scanning-rest/README.md

```diff
@@ -0,0 +1,138 @@
+# Azure Purview Scanning Rest-Level client library for JavaScript
+
+Azure Purview Scanning is a fully managed cloud service whose users can scan your data into your data estate (also known as your **catalog**). Scanning is a process by which the catalog connects directly to a data source on a user-specified schedule.
+
+- Scan your data into your catalog
+- Examine your data
+- Extract schemas from your data
+
+**Please rely heavily on the [service's documentation][scanning_product_documentation] and our [Rest client docs][rest_client] to use this library**
+
+[Source code][source_code] | [Package (NPM)][scanning_npm] | [API reference documentation][scanning_ref_docs]| [Product documentation][scanning_product_documentation]
+
+## Getting started
+
+### Currently supported environments
+
+- Node.js version 13.x.x or higher
+
+### Prerequisites
+
+- You must have an [Azure subscription][azure_subscription] and a [Purview][purview_resource] to use this package.
+
+#### Create a Purview Resource
+
+Follow [these][purview_resource] instructions to create your Purview resource
+
+### Install the `@azure-rest/purview-scanning` package
+
+Install the Azure Purview Scanning client library for JavaScript with `npm`:
+
+```bash
+npm install @azure-rest/purview-scanning
+```
+
+### Create and authenticate a `PurviewScanning`
+
+To use an [Azure Active Directory (AAD) token credential][authenticate_with_token],
+provide an instance of the desired credential type obtained from the
+[@azure/identity][azure_identity_credentials] library.
+
+To authenticate with AAD, you must first `npm` install [`@azure/identity`][azure_identity_npm] and
+[enable AAD authentication on your Purview resource][enable_aad]
+
+After setup, you can choose which type of [credential][azure_identity_credentials] from `@azure/identity` to use.
+As an example, [DefaultAzureCredential][default_azure_credential]
+can be used to authenticate the client:
+
+Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables:
+AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET
+
+Use the returned token credential to authenticate the client:
+
+```typescript
+import PurviewScanning from "@azure-rest/purview-scanning";
+import { DefaultAzureCredential } from "@azure/identity";
+const client = PurviewScanning(
+  "https://<my-account-name>.scanning.purview.azure.com",
+  new DefaultAzureCredential()
+);
+```
+
+## Key concepts
+
+### Rest Client
+
+This client is one of our Rest clients. We highly recommend you read how to use a Rest client [here][rest_client].
+
+## Examples
+
+The following section shows you how to initialize and authenticate your client, then list all of your data sources.
+
+- [List All Data Sources](#list-all-data-sources "List All Data Sources")
+
+### List All Data Sources
```

Comment:
seems like we need more hero scenarios than just listing, since I imagine you'd want to show scanning, etc

Reply:
I believe that the service team will assist us with writing some sample additional samples